components/count-badge/demo/count-badge.html

```diff
@@ -17,20 +17,27 @@
 			<h2>Small Notification Badge</h2>
 			<d2l-demo-snippet>
 				<template>
-					<d2l-count-badge size="small" type="notification" number="1"></d2l-count-badge>
-					<d2l-count-badge size="small" type="notification" number="10"></d2l-count-badge>
-					<d2l-count-badge size="small" type="notification" number="100"></d2l-count-badge>
+					<d2l-count-badge size="small" announce-changes tab-stop text=" 1 new notification." type="notification" number="1"></d2l-count-badge>
```

Comment:
Just an idea, but might be nice to add a button to this demo that increments this number and updates the text to test the live-region